doc/run_settings.rst

```diff
@@ -0,0 +1,159 @@
+that `run_command` will be created and returned.
+If the `run_command` is passed a recognized run command (ex. 'Slurm') the ``RunSettings``
+instance will be a child class such as ``SrunSettings``.
+If not supported by smartsim, the base ``RunSettings`` class will be
+created and returned with the specified `run_command` and `run_args` evaluated literally.
+You may also specify `mpi_run` or `mpi_exec` to the `run_command` argument. This will return
+the associated child class.
+
+Once a ``RunSettings`` object is initialized, you have access to helper functions that
+further configure the run settings of a job. The following chart shows helper functions
+associated with the ``SrunSettings`` object:
+
+.. list-table:: SrunSettings Helper Functions
+   :widths: 25 55 25
+   :header-rows: 1
+
+   * - ``SrunSettings`` func
+     - Example
+     - Description
+   * - ``set_nodes()``
+     - ``SrunSettings.set_nodes(nodes)``
+     - Set the number of nodes
+   * - ``set_tasks()``
+     - ``SrunSettings.set_tasks(tasks)``
+     - Set the number of tasks for this job
+   * - ``set_tasks_per_node()``
+     - ``SrunSettings.set_tasks_per_node(tasks_per_node)``
+     - Set the number of tasks for this job
+   * - ...
+     - ...
+     - ...
+
+=====
+Local
+=====
+When running SmartSim on laptops and single node workstations, the base
+``RunSettings`` object is used to parameterize jobs.
+RunSettings include a `run_command` parameter for local launches
+that utilize a parallel launch binary like `mpirun`, `mpiexec`, and others.
+If no `run_command` is specified, the executable will be launched locally.
+
+The local `launcher` supports the base :ref:`RunSettings API <rs-api>`
+which can be used to run executables as well as run executables
+with arbitrary launch binaries like `mpiexec`. The local launcher
+is the default launcher for all ``Experiment`` instances.
+
+In the following example, we walk through creating and launching locally a
+``Model`` entity with a ``RunSettings`` object.
+First specify the `launcher` argument as `"local"`
+when initializing the ``Experiment``:
+
+.. code-block:: python
+
+      exp = Experiment("name-of-experiment", launcher="local")
+
+Next, we initialize a simple ``Model`` entity that echos `Hello World` to stdout.
+The ``RunSettings`` base object will be created since we set `launcher="local"` when
+initializing the experiment.
+
+.. code-block:: python
+
+      run_settings = exp.create_run_settings("echo", exe_args="Hello World")
+
+Finally, create a ``Model`` using ``Experiment.create_model()``. Specify
+the ``RunSettings`` object along with a `name` for the model:
+
+.. code-block:: python
+
+      model = exp.create_model("hello_world", run_settings)
+
+Next, launch the Model using ``Experiment.start()``:
+
+.. code-block:: python
+
+      exp.start(model)
+
+In this experiment, there is no need to use the ``Experiment.stop()`` function
+since there is no launched orchestrator.
```

Comment:
bump on duped "no need for stop" just in case we can re-use/import this blurb from a single .rst